scss/main.scss

```diff
@@ -86,6 +86,22 @@ $animation-length: 80s;
   }
 }
 
+.logo {
+  background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
```

Comment:
hmm do you think it looks out of place before the white background shows up? we could potentially only add this background attribute in after the white background shows up by styling it in the "scrolled" class. idk what are your thoughts?

Reply:
yep def does, i meant to leave a comment abt it lol -- colors are too harsh in contrast to the pastel bg